Neos.Flow/Classes/Mvc/Routing/UriBuilder.php

```diff
@@ -383,8 +354,6 @@ public function build(array $arguments = [])
         $httpRequest = $this->request->getHttpRequest();
         if ($this->createAbsoluteUri === true) {
             $uri = $httpRequest->getBaseUri() . $uri;
-        } elseif (!$this->createRelativePaths) {
```

Comment:
Guess this just needs to be changed to
```
} else {
    $uri = $httpRequest->getScriptRequestPath() . $uri;
```
instead of removing the whole elseif - at least if I'm correct in assuming the intended behaviour is to keep the previous default behaviour.